src/button-split.scss

```diff
@@ -76,4 +76,15 @@ $fd-button-split-margin: 0.125rem / 2;
       }
     }
   }
+
+  .#{$fd-namespace}-button__text {
+    @include fd-ellipsis();
```

Comment:
Shouldn't this have a reset?

Reply:
Ok, I understand what you did. You don't need the reset coz it's `.fd-button__text`, but if this is the case, you can't just use it here. Split button is a separate component from Button and this violates the self-contained rule. You will need a modifier class for these rules

Reply:
fixed